tests/golden/check-anchors/config.yaml

```diff
@@ -0,0 +1,21 @@
+# SPDX-FileCopyrightText: 2022 Serokell <https://serokell.io>
```

Comment:
I guess we can just reuse the existing default `tests/configs/github-config.yaml`, to avoid having to create too many config files.

Reply:
Or maybe don't use a config file at all, omit the `-c` parameter and let xrefcheck use the default config.